### PR TITLE
Feature/new relic

### PR DESCRIPTION
**Pull request by lighthurst**
Hi @i-mckinney. Please review PR. Check out below Trello tickets. Thanks.

https://trello.com/c/rnUxiGqV/12-sign-up-for-and-analzye-throughput-in-newrelic-s
https://trello.com/c/lE6zFxRG/14-refactor-get-queries-into-one-query-s
https://trello.com/c/NlkvW8It/13-implement-redis-cache-sm
https://trello.com/c/1VRzKuVd/15-node-clustering-sm
https://trello.com/c/pI8abw6j/16-unit-test-crud-routes

### CODE REVIEW

File: server/application.js [hunk 1]
```diff
@@ -1,27 +1,19 @@
 const controller = require('./controller');
 const express = require('express');
-const logger = require('./logger');
-const morgan = require('morgan');
+// const logger = require('./logger');
```
Comment (i-mckinney) [inline server/application.js:3]: delete

Comment (i-mckinney) [reply server/application.js:3]: Morgan too

**Review by i-mckinney — APPROVED** [review]
Hey Aaron, great work. Just remove your bundle and csv from github!

**Comment by lighthurst** [comment]
Thanks Ian!

**Pull request merged**